src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Parser {
+    public static String[] inputParser(String input) {
+        String[] processed = input.split(" ", 2);
```

Comment:
```suggestion
        String[] processedInputs = input.split(" ", 2);
```
Similarly, it may be more obvious that the variable represents a collection if it is named in plural form